src/HtmlReader/HtmlReaderContent.tsx

```diff
@@ -19,7 +19,7 @@ const HtmlReaderContent: React.FC<{
            * content. In paginated mode, however, we must have this set because
            * we have to decide how big the content should be.
            */
-          height: shouldGrow ? 'initial' : height,
```

Comment:
I am not sure if this is needed anymore? When I am on scrolling mode with a big page and click on the TOC button, the popup would just jump to the top of the page blocking the header.

Reply:
I'm not sure if this is needed, but if it isn't, I think the `shouldGrow` should be removed rather than commented out.